Mathlib/Analysis/LocallyConvex/AbsConvex.lean

```diff
@@ -305,3 +305,11 @@ theorem totallyBounded_absConvexHull (hs : TotallyBounded s) :
   exact ⟨hs, totallyBounded_neg hs⟩
 
 end
+
+lemma zero_mem_absConvexHull {s : Set E} [SeminormedRing 𝕜] [NormOneClass 𝕜]
+    [AddCommGroup E] [Module ℝ E] [Module 𝕜 E] [Nonempty s] : 0 ∈ absConvexHull 𝕜 s := by
+  obtain ⟨w, hw⟩ := (inferInstance : Nonempty s)
+  rw [← add_neg_cancel ((1/2 : ℝ) • w), ← smul_neg]
+  exact convex_absConvexHull (subset_absConvexHull hw)
+    ((Balanced.neg_mem_iff balanced_absConvexHull).mpr (subset_absConvexHull hw))
+    (le_of_lt one_half_pos) (le_of_lt one_half_pos) (add_halves 1)
```

Comment:
`absConvexHull` is balanced, so you could use that 0 belongs to any nonempty balanced set (which seems to be a missing lemma):
```lean
lemma Balanced.zero_mem {E 𝕜 : Type*} [SeminormedRing 𝕜] [AddCommMonoid E] [Module 𝕜 E] {s : Set E}
    (hs_nonempty : s.Nonempty) (hs : Balanced 𝕜 s) : 0 ∈ s := by
  have : (0 : E) = (0 : 𝕜) • (hs_nonempty.some : E) := by simp
  rw [this]
  exact hs.smul_mem (a := 0) (by simp) hs_nonempty.some_mem
```

Reply:
There is `balancedCore_zero_mem` and `balancedCore_nonempty_iff` with slightly different hypothesis.